core/storage/feedback/gae_models.py

```diff
@@ -179,21 +243,40 @@ def create(cls, exploration_id, thread_id, message_id):
 
     @classmethod
     def get(cls, exploration_id, thread_id, message_id, strict=True):
-        """Gets the FeedbackMessageModel entry for the given ID.
-
-        If the message id is valid and it is not marked as deleted, returns the
-        message instance. Otherwise:
-        - if strict is True, raises EntityNotFoundError
-        - if strict is False, returns None.
+        """Gets the FeedbackMessageModel entry for the given ID. Raises error if
+            no undeleted messages are found and strict == True.
+
+        Args:
+            exploration_id: str. ID of the exploration
+            thread_id: str. ID of the thread.
+            message_id: int. ID of the message.
+            strict: bool. Whether to raise an error if no FeedbackMessageModel
+                entry is found for message_id.
+
+        Returns:
+            None, if strict == False and no undeleted message with the given
```

Comment:
Reword to put all the types first, e.g. `FeedbackMessageModel or None. If ...., returns None. Otherwise, returns the FeedbackMessageModel corresponding to the given IDs.`